.gitignore

```diff
@@ -15,7 +15,8 @@ setup.cfg
 */*.pyc
 *~
 
-node_modules/
+/node_modules
+/package-lock.json
```

Comment:
Note: package-lock.json should be in the git repository.

See
* https://stackoverflow.com/questions/44297803/what-is-the-role-of-the-package-lock-json
* https://docs.npmjs.com/files/package-lock.json

For now, there is no package-lock.json in the respository, so package-lock.json in .gitignore makes ```git add``` easier.

Reply:
You mean, you want to see them as "unversioned" files .. right?

Reply:
If so, should I also remove them from the "clean" target https://github.com/asciimoo/searx/pull/1900/files#diff-b67911656ef5d18c4ae36cb6741b7965R112-R114

Reply:
The target is to have them in the repository, but no big bang : searx exists without them since the beginning.

Yes unversioned file and ```make node.clean``` can remove them for now.

Reply:
OK

> The target is to have them in the repository,

I know these concepts from other package managers but I have a doubt that it make sense to fix all requirements .. in practice no one takes a regularly look at all the vulnerabilities and the software is build a the time with all the vulnerabilities .. but here is not the right place to discuss this aspect  :)